tables/isolates.d/takashita22c-iso.csv

```diff
@@ -0,0 +1,4 @@
+﻿iso_name,var_name,site_directed,gisaid_id,genbank_accn,sra_accn,expandable
+hCoV-19/USA/NY-MSHSPSP- PV56475/2022,Omicron/BA.2.12.1,FALSE,13512442,NULL,NULL,TRUE

```

Comment:
@zachandfox, is it an extra space in the `iso_name` ?

Reply:
That is how it is presented on GIS, I forgot to ask if we should remove it anyway

Reply:
OK, let's use the way they write it.